### ISSUE TITLE
In JS interop, add Invoke/InvokeAsync overloads that use C#11 param span

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#40098]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

Hopefully we can save one allocation per call, at least in the cases that had to use `params` in the past.

### Describe the solution you'd like

-

### Additional context

-

**Comment by SteveSandersonMS** [comment]
@mu88 Oh actually I see your point that the language feature itself isn't there yet. In that case it's fairly clear that it wouldn't be something we could do for Blazor!

**Comment by mu88** [comment]
Yeah that's what I mean @SteveSandersonMS - I'm happy to assist as soon as the language feature is available

**Comment by WeihanLi** [comment]
Is there an approved API proposal?

**Comment by camkinney** [comment]
It looks like this C# language feature was original proposed on this Issue: https://github.com/dotnet/csharplang/issues/1757
It has now been subsumed by this Proposal issue for Parent Collections support in general https://github.com/dotnet/csharplang/issues/7700

**Comment by javiercn** [comment]
Will use https://github.com/dotnet/aspnetcore/issues/62312 to track changes in this area
